Changes in module 'nf-core/sniffles'
--- modules/nf-core/sniffles/meta.yml
+++ modules/nf-core/sniffles/meta.yml
@@ -1,8 +1,10 @@
 name: sniffles
 description: structural-variant calling with sniffles
 keywords:
-  - sniffles
+  - sniffles c x x 
+  - sniffles a v c 
   - structural-variant calling
+  - deluxe
 tools:
   - sniffles:
       description: a fast structural variant caller for long-read sequencing

--- modules/nf-core/sniffles/main.nf
+++ modules/nf-core/sniffles/main.nf
@@ -3,9 +3,9 @@
     label 'process_high'
 
     conda "bioconda::sniffles=2.0.7"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    container '${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/sniffles:2.0.7--pyhdfd78af_0' :
-        'biocontainers/sniffles:2.0.7--pyhdfd78af_0' }"
+        'biocontainers/sniffles:2.0.7--pyhdfd78af_0' }'
 
     input:
     tuple val(meta), path(bam), path(bai)

************************************************************
